[Bug]: API incompatible Typescript type for aqlQuery

### Verified issue does not already exist?

- [x] I have searched and found no existing issue

### What happened?

There seems to be a mismatch between the type definitions in

`/node_modules/@actual-app/api/@types/loot-core/src/shared/query.d.ts` and
`/node_modules/@actual-app/api/@types/app/query.d.ts`

which causes the following tsc compiler error for me:

```
src/sync.ts(114,9): error TS2345: Argument of type 'Query' is not assignable to parameter of type 'import("/app/node_modules/@actual-app/api/@types/loot-core/src/shared/query").Query'.
  Type 'Query' is missing the following properties from type 'Query': reset, serializeAsString
The command '/bin/sh -c npm run build' returned a non-zero code: 2
```

### How can we reproduce the issue?

How can we reproduce the issue?

With version `"@actual-app/api": "^25.12.0"` try to compile this code:

```typescript
import { q, aqlQuery } from "@actual-app/api";

await aqlQuery(
  q("transactions")
    .filter({ imported_id: tx.imported_id })
    .select(["id", "amount"])
);
```

### Where are you hosting Actual?

Other

### What browsers are you seeing the problem on?

Other

### Operating System

Linux

--- a/packages/api/app/query.js
+++ b/packages/api/app/query.js
@@ -97,6 +97,14 @@ class Query {
   serialize() {
     return this.state;
   }
+
+  reset() {
+    return q(this.state.table);
+  }
+
+  serializeAsString() {
+    return JSON.stringify(this.serialize());
+  }
 }
 
 export function q(table) {

--- a/upcoming-release-notes/6331.md
+++ b/upcoming-release-notes/6331.md
@@ -1,0 +1,6 @@
+---
+category: Bugfix
+authors: [BHChen24]
+---
+
+Updates the `Query` class in `packages/api/app/query.js` to include the missing `reset()` and `serializeAsString()` methods.

